README.adoc

```diff
@@ -162,16 +162,16 @@ include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClien
 ----
 This class is a client for the `system` service.
 
-In this class there is a method called `getPropertiesHelper()`. This method makes a request to the
-`system` service, and checks for the response. If a `Status.OK`` response which is corresponding to the
+A method called `getPropertiesHelper()` is included in this class. This method makes a request to the
+`system` service and checks for the response. If a `Status.OK` response that corresponds to the
 200 status code is found, the method returns the system properties from the `system` service request.
-In this guide, a new if statement has been added to the method to check if the response is specifically
-a `Status.SERVICE_UNAVAILABLE` that is a 503 status code. When this happens, the server being called
-is unable to handle the request due to a temporary overload or scheduled maintenance, which would
-likely be alleviated after some delay. To simulate system being unavailable, an IOException is thrown.
+In this guide, a new `if` statement is added to the method to check if the response is specifically
+a `Status.SERVICE_UNAVAILABLE` message, which is a 503 status code. This code indicates that the server being called
+is unable to handle the request because of a temporary overload or scheduled maintenance, which would
+likely be alleviated after some delay. To simulate that the system is unavailable, an IOException is thrown.
 
-This `getPropertiesHelper()` method is called by the public `getProperties()` method which is used 
-in `InventoryManager.java`. You will look into this class in more detail in the next section.
+The public `getProperties()` method, which is used 
```

Comment:
=> "The public `getProperties()` method, which is in the `InventoryManager.java` class,"